test/swap.js

```diff
@@ -32,8 +29,8 @@ describe('swap', function() {
   });
 
   it('swaps property values from one property to another', function() {
-    eq(R.swap('a', 'b', o), {a: 2, b: 1});
-    eq(R.swap('b', 'a', o), {a: 2, b: 1});
+    eq(R.swap('a', 'b', {a: 1, b: 2}), {a: 2, b: 1});
+    eq(R.swap('b', 'a', {a: 1, b: 2}), {a: 2, b: 1});
```

Comment:
Colocating input & output to make it easier to understand how things are connected.